SCF-Session-4/clumps/ArrayClumps.java

```diff
@@ -0,0 +1,46 @@
+package clumps;
+
+/**
+ * Contains a method to calculate the number of clumps in an array.
+ * @author Ishita Mathur
+ *
+ */
+public class ArrayClumps 
+{
+	/**
+	 * This method returns the number of clumps in an array.
+	 * @param inputArray, should contain a series of 2 or more adjacent elements of the same value
+	 * @param arrayLength > 0
+	 * @return the count of clumps in the array.
+	 */
+	public int countClumps(int[] inputArray, int arrayLength)
+	{
+		int occur = 0, count = 0;
+		
+		if (arrayLength == 0)
+			throw new AssertionError("Array is empty");
+		
+		for(int i = 0; i < arrayLength; i++)
+		{
+			for(int j = i + 1; j < arrayLength; j++)
+			{
```

Comment:
Nested loop is not really needed to resolve this. Try to resolve with single loop.